components/wizard/ProjectSelector.vue

```diff
@@ -135,14 +135,14 @@
         <div
           v-if="
             canCreateProjectPermissionCheck &&
-            !canCreateProjectPermissionCheck.authorized
+            !canCreateProjectPermissionCheck.authorized &&
+            showUpgradeButton
```

Comment:
this naming initially confused me. probably would benefit from a refactor to `showUpgradePlanButton` or something similar to make a distinction between **software upgrade** and **plan upgrade**. but we wouldn't be checking software versions in `ProjectSelector.vue` anyways so this doesn't actually matter